src/SFA.DAS.Support.Shared/SearchIndexModel/UserSearchModel.cs

```diff
@@ -18,5 +18,7 @@ public class UserSearchModel : BaseSearchModel
         public string FirstNameSearchKeyWord { get => FirstName.ToLower(); }
         public string LastNameSearchKeyWord { get => LastName.ToLower(); }
         public string NameSearchKeyWord { get => Name.ToLower(); }
+        public bool IsSuspended { get; set; }
```

Comment:
any need for this change? The new property isn't being used anywhere

Reply:
It is consumed through a nuget package in the employerUser -> SFA.DAS.EmployerUsers.Support.Web. I did this to expose this field to be consumed in future. Hopefully I understood correctly when were talking about this :)